app/models/manageiq/providers/nuage/manager_mixin.rb

```diff
@@ -17,7 +17,7 @@ def raw_connect(username, password, endpoint_opts)
     end
 
     def auth_url(protocol, server, port, version)
-      scheme = protocol == "ssl-with-validation" ? "https" : "http"
+      scheme = !protocol || (protocol == "non-ssl") ? "http" : "https"
```

Comment:
I find this `!protocol` part confusing*, perhaps we would be better off with:
```
scheme = protocol.to_s.start_with? "ssl" ? "https" : "http"
```

*\*Too much Java-like. I mean, we're in ruby, we shouldn't check for `null` all over the code :wink:*

Reply:
You mean `nil` ? :)